### ISSUE TITLE
Removing a property during livecoding does not restore it to default value - it remembers the last livecoded value. 

### ISSUE THREAD

**Issue by StijnKuipers** [makepad/makepad#387]

**Comment by kevinaboos** [comment]
yeah, i also find this behavior a bit confusing sometimes. Would be nice to have default values applied.

**Comment by StijnKuipers** [comment]
This seems to have been addressed now - not sure if it is 100% tested through but it seems to basically work.

**Comment by kevinaboos** [comment]
yea this was fixed in https://github.com/makepad/makepad/commit/d8a4f24253ade7ca9af57af058858d74c8507740
